schema.json

```diff
@@ -0,0 +1,392 @@
+[
+  {
+    "name": "consensusTimestampTruncated",
+    "type": "TIMESTAMP",
+    "description": "consensusTimestamp truncated to microseconds. Used for partitioning."
```

Comment:
`"mode": "REQUIRED"` could be added here and to other required fields.

Reply:
This was the first cut.
https://github.com/hashgraph/hedera-mirror-node/blob/blockchain-etl/hedera-mirror-importer/src/main/resources/schema1.json - well thought `REQUIRED`s. 
There are so many fields! .. I wasn't sure about many of them.
So removed them from first cut to keep things simple - basically avoid failures 10 days later only to find a dumb labelling. :D

Reply:
> basically avoid failures 10 days later only to find a dumb labelling. :D

In regards to this, it might be a good idea to also dump all data coming from Pub/Sub to a GCS bucket (can be done with a Dataflow template https://cloud.google.com/dataflow/docs/guides/templates/provided-streaming#cloudpubsubtogcstext). So that in case we find bugs or need to change something, this data can be quickly reprocessed from GCS in batch mode.

Reply:
Sounds like a good idea.